### DISCUSSION TITLE
Rack type

### DISCUSSION THREAD

**Discussion by Hunterkid2222** [janstehno/cotw-companion#20] [Ideas]
They should include a brief thing about all that animals rack types

**Comment by janstehno** [comment]
Like trophy type? If that is the case, animals with antlers/horns/tasks have them as trophy type, otherwise it is weight/length. It would be pointless to add this. Otherwise I have no idea what exactly you have in mind.

**Reply by Hunterkid2222** [reply]
There are diffrent racks that can give u a different rating for most rack animals

**Reply by janstehno** [reply]
Well, as it is a random rack generator that does this, there is eventually infinite number of them. No way I can implement this. Sorry.
